#Bài 07: Đoạn chương trình sau in ra gì?
 #   number = 5.0
 #   try:
 #       r = 10/number
  #      print(r)
  #  except:
  #      print("Oops! Error occurred.")
  =>>>2.0



#Bài 08: Đoạn chương trình sau thực viện việc gì?
#    try:
        # đoạn code có thể gây ra lỗi
#    except (TypeError, ZeroDivisionError):
#        print("Python Quiz")
khi gán giạ trị hay thục hiện nhiêm vụ sai sẽ thực hiện typeerror




#Bài 09: Kết quả output của đoạn chương trình sau là gì?
#    def hoan_function():
 #       try:
 #           print('Monday')
 #       finally:
  #          print('Sunday')
in ra : Monday
        Sunday
        None




#Bài 10: Kết quả của đoạn chương trình sau là gì?
  #  try:
  #      print("throw")
  #  except:
  #      print("except")
 #   finally:
#        print("finally")
in ra:
     throw
     finally

